scripts/ChangesSinceLastRelease.txt

```diff
@@ -188,3 +188,33 @@ Windows.Win32.System.Mmc.View.get_CellContents : CellContents...UInt16** => PBST
 Windows.Win32.System.Mmc.View.get_Memento : Memento...UInt16** => PBSTR
 Windows.Win32.System.Ole.Apis.QueryPathOfRegTypeLib : lpbstrPathName...UInt16** => PBSTR
 Windows.Win32.System.Ole.Apis.VarBstrCat : pbstrResult...UInt16** => PBSTR
+# Untangle window scroll/status enums and flags
+Windows.Win32.Media.DirectShow.IVideoWindow.get_WindowState : WindowState...Int32* => SHOW_WINDOW_CMD*
+Windows.Win32.System.Ole.IPropertyPage.Show : nCmdShow...UInt32 => SHOW_WINDOW_CMD
+Windows.Win32.System.SystemServices.SHOW_WINDOW_STATUS added
```

Comment:
We shouldn't be adding types to SystemServices. Either add an explicit namespace to the enum definition or add a usage so the namespace is inherited.

Reply:
Good catch!